### PR TITLE
fix: collection on mobile is broken

### LINKED ISSUE #5448: Collection on mobile 📱 is broke
Seems overflowing this?

https://user-images.githubusercontent.com/5887929/229130848-5599c2ea-7b37-47ad-90ed-7122dc89bb9e.mp4

second container renders too big?

https://user-images.githubusercontent.com/5887929/229131093-7ed0f8d9-b1d5-44e0-a466-a85a9e94c626.mp4

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5448
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image
image

### CODE REVIEW

File: components/collection/CollectionInfo.vue [hunk 1]
```diff
@@ -119,6 +119,10 @@ const { stats } = useCollectionDetails({ collectionId: collectionId.value })
     flex-direction: column;
   }
 
+  .mobile-column-padding-bottom {
+    padding-bottom: 0;
+  }
+
```
Comment (vikiival) [inline components/collection/CollectionInfo.vue:125]: Should not be there some `@media` ?

Just asking

**Review by daiagi — CHANGES_REQUESTED** [review]
you broke the design.
it needs to remain 2 columns on mobile.

**Comment by Jarsen136** [comment]
> this brakes the design. it needs to remain 2 columns on mobile.
> 
> from figma:
> 
> image

For 2 columns struct, the text would overflow out of the screen. I guess placing them in one column on mobile could contain more text content. WDYT @exezbcz 
image

**Comment by exezbcz** [comment]
@Jarsen136 yes. that is better, I made it like that to make it simpler to convert the desktop design to mobile.

In this case I dont mind that, but thanks for noticing! @daiagi

**Review by daiagi — APPROVED** [review]

**Comment by roiLeo** [comment]
Tested on pixel

Before




AFTER

File: components/collection/CollectionInfo.vue [hunk 2]
```diff
@@ -19,15 +19,15 @@
         @click.native="toggleSeeAllDescription" />
     </div>
     <div>
-      <div class="columns is-mobile">
-        <div class="column">
+      <div class="columns pt-3">
```
Comment (roiLeo) [inline components/collection/CollectionInfo.vue:22]: I wouldn't add padding on columns/column, wrap it in a new div instead

**Comment by vikiival** [comment]
Sooo last fixes and LG (let's go/ life's good)(choose for yourself) 🚀

File: components/collection/CollectionInfo.vue (hunk 2 above)
Comment (Jarsen136) [reply components/collection/CollectionInfo.vue:22]: In this component, I need to overwrite some padding on `column`, so I guess I wrap it in a new div. wdyt

Comment (roiLeo) [reply components/collection/CollectionInfo.vue:22]: Please don't add spacing in columns/column class. If you want to add spacing then create a inner div that wrap content.
If you don't want spacing you can use [gapless](https://bulma.io/documentation/columns/gap/#gapless) helper or change columns to flex.

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 32.69 USD/KSM ~ 0.918 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x6a9c557c89c010eb06b670ed682b642500ce4bc3d3df71d2b65bf6485845166c](https://kusama.subscan.io/extrinsic/0x6a9c557c89c010eb06b670ed682b642500ce4bc3d3df71d2b65bf6485845166c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**